robot_controllers/src/cartesian_twist.cpp

```diff
@@ -282,14 +304,21 @@ void CartesianTwistController::command(const geometry_msgs::Twist::ConstPtr& goa
     return;
   }
 
+  std::string frame;
+  frame = goal->header.frame_id;
+  if (frame.empty())
+  {
+    return;
```

Comment:
That sort of logic should really be handled in the teleop -- not the controller.